src/helper/coordinate.rs

```diff
@@ -103,27 +103,27 @@ pub fn coordinate_from_index_with_lock(
 ) -> String {
     format!(
         "{}{}{}{}",
-        if is_lock_col == &true { "$" } else { "" },
+        if *is_lock_col { "$" } else { "" },
         string_from_column_index(col),
-        if is_lock_row == &true { "$" } else { "" },
+        if *is_lock_row { "$" } else { "" },
         row
     )
 }
 
 pub(crate) fn adjustment_insert_coordinate(num: &u32, root_num: &u32, offset_num: &u32) -> u32 {
-    let mut result = *num;
-    if (num >= root_num && offset_num > &0) || (num < root_num && offset_num < &0) {
```

Comment:
the `offset_num` is never less than zero because it is an `u8` therefore this always be false. hence the other part of or is not needed.